python/ml4ir/base/model/cyclic_learning_rate.py

```diff
@@ -0,0 +1,281 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
@mohazahran I don't know if we should keep this copyright.
Thoughts @balikasg

Reply:
Honestly I do not know. In good faith I would even say "Copied from AAA due to version issues."  It implies we respect their license. But I have not read it.

Reply:
Yep. We should cite the source.

Reply:
I agree, that's why I left it. But I will add specific citation.